### DISCUSSION TITLE
Make.com - Deepgram update

### DISCUSSION THREAD

**Discussion by badibadey** [deepgram/community#917] [General help] [answered]
Hi, 

I wanted to invite you to vote to update the deepgram application at make.com. The module “transcribe a pre recorder files from URL” - contains outdated engines. It is missing Whisper and Nova-2 - I think many of us will find this useful in the future, but unfortunately we need votes.

link:
https://www.make.com/en/app-improvement-ideas/p/deepgram-transcribe-a-pre-recorded-audio-file-from-url

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@badibadey  thanks for the suggestion we can look into this improvement.

**Comment by premiumlion** [comment]
Hi, I just started using the integration, and yes. It's extremely outdated for make (models are from Nova-1).

Can you please update the integration, or provide a guide on how to use the API or something with Make.com

Thank you.
